01-js-intro-and-jquery/10/js/scripts.js

```diff
@@ -0,0 +1,50 @@
+$(document).ready(function() {
+	
+	var alias = '';
+	var name  = '';
+	var nameLength = '';
+	var lastChange = '';
+	
+	var focusAlias = function() {	
+		$('.alias').focus(); 
+	};
+
+	var highlightName = function() {
+		name = $('#response').html();
+		name = name.substring(8);
+		nameLength = name.length-1;
+		name = name.substring(0,nameLength);
+		lastChange = $('#response').html().replace( name, "<strong class='bold'>" + name + "</strong>" );
+		$('#response').html(lastChange);
+	};
+	
+	var getResponse = function() {
+		alias = $('.alias').val();
+
+		$.ajax({
+   url: "http://bootcamp.aws.af.cm/welcome/" + alias,
+   type: 'GET',
+   dataType: "json",
+   success: function( data, status, xhr ) {
+    console.log(status);
+   	if(data.error) {
+  			$('#response').addClass('error').html(data.error);
+   	} else {
+   		$('#response').removeClass('error').html(data.response);
+   		highlightName();
```

Comment:
What about passing ``alias`` as an argument to ``highlightName`` and highlight the result based on that? ;)